src/Application/Items/Commands/RepairUserItemCommand.cs

```diff
@@ -44,43 +44,24 @@ public async Task<Result<UserItemViewModel>> Handle(RepairUserItemCommand req, C
                 return new(CommonErrors.UserItemNotFound(req.UserItemId));
             }
 
-            if (userItem.Rank >= 3)
+            if (!userItem.IsBroken) // repair
```

Comment:
The comment is outdated and can be removed

Reply:
fixed